src/components/header/styles.css

```diff
@@ -0,0 +1,26 @@
+.main__header,
+.main__subheader {
+  margin: 0;
+  font-size: 2.625rem;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  line-height: 3.2rem;
+  text-align: center;
+}
+.main__header {
+  padding: 1rem 0.7rem;
+  font-weight: 600;
+}
+
+.main__subheader {
+  padding: 0 1rem;
+  font-weight: 500;
+}
+
+@media only screen and (max-width: 767px) {
```

Comment:
This could be adjusted to the mobile first rule so I suggest switching the styles here with their counterparts above and replacing the screen rule with: `@media screen and (min-width: 768px)`.